libnopegl/src/node_texture.c

```diff
@@ -412,6 +412,11 @@ static int handle_media_frame(struct ngl_node *node)
     ngli_image_reset(&s->image);
 
     int ret = ngli_hwmap_map_frame(&s->hwmap, frame, &s->image);
+
+    /* Signal image change on new frame */
+    s->image_rev++;
+    s->image.rev = s->image_rev;
```

Comment:
Shouldn't it be the other way around?
```c
s->image.rev++; // signal image change on new frame
s->image_rev = s->image.rev; // consider the revision immediately honored
```
?

Reply:
Changed to `s->image_rev = s->image.rev++;`